.github/workflows/rust.yml

```
@@ -9,42 +9,42 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
```

any reason not to use the latest release?
```suggestion
    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```